### DISCUSSION TITLE
Update monterey to 12.3

### DISCUSSION THREAD

**Discussion by tommaso993** [SkyrilHD/Dell-E7250-Hackintosh#53] [General]
hi i need some information, thanks to your work i was able to easily install monterey a couple of months ago and now i wanted to know if i can upgrade to monterey 12.3 from 12.2
Thank you

**Comment by SkyrilHD** [comment]
Sure. You should be able to update without an issue, unless Apple changed something that breaks it.

**Reply by tommaso993** [reply]
Haven't you updated yet?
Anyway, thank you for the answer

**Reply by SkyrilHD** [reply]
I am on 12.3 developer beta and nothing is broken yet

**Reply by tommaso993** [reply]
Perfect thank you bro

**Reply by tommaso993** [reply]
i upgraded to monterey 12.3 with your upgraded efi and everything works perfectly
thanks again for your work
